xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/VaapiEGL.h

```diff
@@ -31,6 +33,10 @@
 #include <EGL/egl.h>
 #include <EGL/eglext.h>
 #include <va/va.h>
+#include <va/va_drmcommon.h>
+
+#include "guilib/Geometry.h"
```

Comment:
this dependecy to guilib is kind of ugly. eventually VP should be used transcoding on a headless system without a gui. I don't see much benefit in using this CSizeInt class here.

Reply:
ofc it's not strictly necessary - for me the benefit here is that they encapsulate the return value in one atomic unit. Otherwise you'd have to have two functions for width and height, but 100% of the time you need both anyway. Or pass references as out arguments but that's also kind of ugly. 
Actually, I don't see why CSizeInt etc. should be in guilib at all. Going forward I think it would actually be good to use these classes where appropriate since they encapsulate related data and support useful operations with them. They are not really related or limited to the GUI at all. I use them a lot in the Wayland code already, which is also not quite GUI. Could we move them out of guilib?

Reply:
I don't mind if you move it

Reply:
Is it OK if I make a separate PR for that afterwards? It will be a search-replace job and should not be mixed with the vaapi changes here.

Reply:
ok, if you are going to do this in the near future